.vsts-ci.yml

```diff
@@ -59,3 +59,8 @@ jobs:
   - task: PublishBuildArtifacts@1
     inputs:
       pathtoPublish: $(Build.ArtifactStagingDirectory)
+
+  - task: CmdLine@2
+    displayName: 'Run unit tests'
+    inputs:
+      script: 'dotnet test reference\SimpleCalculator\SimpleCalculator.Tests\SimpleCalculator.Tests.csproj -c Release --collect "XPlat code coverage" --logger trx'
```

Comment:
This must be integrated into the for loop in the ps1 script (after [this line](https://github.com/unoplatform/Uno.Samples/blob/ed53201aebb6a1649a3c231ce6cff0b16a90a66b/.vsts-ci.yml#L45), and using a `*.Tests.dll` wildcard). It will otherwise fail for other projects in the solution (they are built conditionally).

Reply:
@eriklimakc can you action this please?